### ISSUE TITLE
Direct Feedback Alignment Scales to Modern Deep Learning Tasks and Architectures

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#127]
# TL;DR
A study of Direct Feedback Alignment, which allows us to parallelize parameter updates in a variety of tasks, such as the Transformer model, Graph Conv, and recommendation, and they confirm that the results are not bad.

![Table 2 AUC (higher is better) and log loss (lower is better) of recommender systems trained on the](https://user-images.githubusercontent.com/24524018/95158676-31cfd800-07d7-11eb-8375-f26c7664df00.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2006.12878

# Submission Dates(yyyy/mm/dd)
2020/06/23

# Authors and institutions
Julien Launay, Iacopo Poli, François Boniface, Florent Krzakala
- LightOn 
- École Normale Supérieure

# Methods

# Results

# Comments
